### ISSUE TITLE
crash on attempted conversion of json created by 2023.4

### ISSUE THREAD

**Issue by erevrav** [dhmay/jamstats#168]
**Describe the bug**

**To Reproduce**
Windows 11, ran jamstats artifact from Oct 31 (with color fix), feed it json created by crg 2023.4 (technically a fork of it, but I hypothesize that's not relevant). 

**Expected behavior**
expected pdf. got crash:
```
jamstats version 1.4.1, by Damon May
Mode not specified. Inferred mode pdf
Output filepath not provided. Using input filepath with extension .pdf instead of .json
Writing PDF file...
Using theme white
[76744] Failed to execute script 'jamstats' due to unhandled exception!
Traceback (most recent call last):
  File "pandas\core\groupby\generic.py", line 341, in array_func
  File "pandas\core\groupby\ops.py", line 1016, in _cython_operation
  File "pandas\core\groupby\ops.py", line 677, in cython_operation
  File "pandas\core\groupby\ops.py", line 508, in _cython_op_ndim_compat
  File "pandas\core\groupby\ops.py", line 563, in _call_cython_op
  File "pandas\core\groupby\ops.py", line 205, in get_cython_func_and_vals
  File "pandas\core\groupby\ops.py", line 171, in _get_cython_function
NotImplementedError: function is not implemented for this dtype: [how->mean,dtype->object]

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "jamstats", line 255, in <module>
  File "jamstats", line 251, in __main__
  File "jamstats\plots\plot_together.py", line 93, in save_game_plots_to_pdf
  File "jamstats\plots\plot_together.py", line 67, in make_all_plots
  File "jamstats\plots\basic_plots.py", line 46, in plot
  File "jamstats\data\game_data.py", line 239, in build_team_jammersummary_df
  File "pandas\core\groupby\generic.py", line 979, in aggregate
  File "pandas\core\apply.py", line 161, in agg
  File "pandas\core\apply.py", line 436, in agg_dict_like
  File "pandas\core\apply.py", line 436, in <dictcomp>
  File "pandas\core\groupby\generic.py", line 243, in aggregate
  File "pandas\core\groupby\groupby.py", line 1690, in mean
  File "pandas\core\groupby\generic.py", line 352, in _cython_agg_general
  File "pandas\core\groupby\generic.py", line 348, in array_func
  File "pandas\core\groupby\groupby.py", line 1398, in _agg_py_fallback
  File "pandas\core\groupby\ops.py", line 1060, in agg_series
  File "pandas\core\groupby\ops.py", line 1085, in _aggregate_series_fast
  File "pandas\_libs\reduction.pyx", line 297, in pandas._libs.reduction.SeriesGrouper.get_result
AssertionError: `result` has not been initialized.
```

**Additional context**
running CRG from https://github.com/katpet/scoreboard/releases/tag/v2023.4_pbt_v0.3, which is up to date with mainline crg and this is (probably) something far away from those changes.

**Comment by erevrav** [comment]
(the jsons from the other 4 games that weekend work fine with the artifact from Oct 31.)

**Comment by dhmay** [comment]
I believe this is fixed in 1.4.2.
